### PR TITLE
Schedule subview

### PR DESCRIPTION
**Pull request by zmbc**
Superset of #14

### CODE REVIEW

**Review by juimin — CHANGES_REQUESTED** [review]
- Not sure what Moment._d is or if it is something you already accounted for but when I log the start and end passed to the select function, a selection's end._d gives me the correct day but the time is offset by 6.5 hours. start._d gives a 7 hour offset. Can't see how you change this in the code though. If this is irrelevant then you can ignore but the incorrect report seems concerning

File: client/src/Components/Schedule/Schedule.js [hunk 1]
```diff
@@ -1,18 +1,67 @@
...
+      selectable: true,
+      header: {
+        left: '',
+        center: '',
+        right: ''
+      },
+      // Only permit selections within one day
+      selectConstraint:{
+        start: '00:01',
+        end: '23:59',
```
Comment (juimin) [inline client/src/Components/Schedule/Schedule.js]: This ends up making it so 12:00 AM and 11:30 PM are not options. Do these not count as starting or ending within the same day?

**Review by juimin — CHANGES_REQUESTED** [review]
This seems weird but just double checking to make sure this is intentional

File: client/src/Components/Schedule/Schedule.js (hunk 1 above)
Comment (zmbc) [reply client/src/Components/Schedule/Schedule.js]: Very, very good catch. Looks like I can fix by just using 0:00 and 24:00.

**Comment by zmbc** [comment]
> Not sure what Moment._d is or if it is something you already accounted for but when I log the start and end passed to the select function, a selection's end._d gives me the correct day but the time is offset by 6.5 hours. start._d gives a 7 hour offset. Can't see how you change this in the code though. If this is irrelevant then you can ignore but the incorrect report seems concerning

Can you explain what you mean by this? Here's how I verified that it was working:
calendar
request

**Review by juimin — APPROVED** [review]

**Pull request merged**